### PR TITLE
313: For educational purposes

### PR DESCRIPTION
**Pull request by pimotte**
習おう。Let's learn!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -194,6 +194,10 @@ request dated after any votes resets those votes.
 If the rule-change is adopted, a player who can shall merge the pull-request
 in a timely fashion, which marks completion of the vote.
 
+**313** Any vote through a comment on a pull request is invalid, unless one of the
+comments above the vote contains a word or sentince in either Japanese or Turkish
```
Comment (MrHug) [inline README.md]: sentice => sentence

Comment (pimotte) [inline README.md]: Oops.

**Comment by MrHug** [comment]
Not sure if @arthurbik ever picked up Japanese again (he did start it, iirc?), but he might not like this rule much ;)

Also can we add to the rule that it must be unique, i.e. we can not just copy your message from this PR every time?

**Comment by pimotte** [comment]
I specifically formulated it as such that another player can comment with a message and it doesn't have to be the initiator of the PR.

Edit: Also, fixed uniqueness. I considered that, but let it go in the first place, but if you'd like it, I have nothing against it.

**Comment by MrHug** [comment]
Alright, well I will abstain from voting for this (though I am in favour) to ensure that @arthurbik gets a deciding vote ;)

**Comment by arthurbik** [comment]
Japanese or Turkish, that's specific... I can't say I have become any better, but I will one day learn Japanese. So I like this idea.

What happens when the Japanese/Turkish or translation contains a mistake?

**Comment by arthurbik** [comment]
Ahh, and thanks @MrHug

**Comment by pimotte** [comment]
I purposefully chose "a translation", so I'd suggest something that is in some way defensible as a translation is okay. If it's just blatently incorrect (ie. not a spelling error), then I guess a #18 might be in order. If that gets adopted.

Edit: If you'd like me to I could add something in that explicitly states it. Also, the Turkish is because Jesse is learning that currently.

**Comment by arthurbik** [comment]
But who broke a rule when the initiator of the PR is not the one who wrote the Japanese/Turkish + translation?

**Comment by pimotte** [comment]
Mmm. In it's current formulation, it would be the person who merges the PR. (Since casting an invalid vote shouldn't really be a problem, but adopting rule-change without a valid majority would.) I'm tempted to make this say "possibly incorrect translation", but that enables abuse in the other direction.

**Comment by Jason-Statham** [comment]
test

**Comment by arthurbik** [comment]
Other players may suggest amendments or argue against the proposal before the vote. A reasonable time must be allowed for this debate.

There are no sanctions for voting when this rule disallows it, regardless of what other rules specify. 

Isn't this a conflict?

**Comment by pimotte** [comment]
By 110, unmutable rules always supersede mutable rules. Hence the "no sanctions" clause can only apply to other mutable rules. (So it will result in a conflict, but the resolution-rules state that the rule you quote must be used.) It's mostly a pre-measure against 311, because I want this rule to be for fun and education, and not disrupt the rest of the game. 

I do see how you're reading it though, so I'll add in a change that will make this less wide of a clause.

File: README.md [hunk 2]
```diff
@@ -195,6 +195,13 @@ request dated after any votes resets those votes.
 If the rule-change is adopted, a player who can shall merge the pull-request
 in a timely fashion, which marks completion of the vote.
 
+**313** Any vote through a comment on a pull request is invalid, unless one of the
+comments above the vote contains a word or sentence in either Japanese or Turkish 
+and a translation in English. Uniqueness of this word or sentence is encouraged,
+as is correctness of the translation. There are no sanctions for voting
+when this rule disallows, it regardless of what other rules specify,
```
Comment (MrHug) [inline README.md]: I am not sure why there is a comma after "disallows"?

**Comment by pimotte** [comment]
Fixed.

**Comment by pimotte** [comment]
:+1:

**Comment by MrHug** [comment]
:+1:

**Comment by arthurbik** [comment]
Note that this is after #22

**Comment by arthurbik** [comment]
:+1:

**Comment by MrHug** [comment]
Hmm I see I forgot that I would not vote on this. As @arthurbik is in favour regardless, I guess it doesn't matter.

**Comment by pimotte** [comment]
With 3 votes in favor, among 4 players, a simple majority has been achieved, so by rule 203, the suggested rule-change is adopted. As prescribed by rule 314, I will merge this pull request, since the last commit satisfies the exception mentioned in the rule.

**Pull request merged**